MonoDevelop.MSBuild/Schemas/AspireAppHost.buildschema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "license": "Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.",
+  "properties": {
```

Comment:
Would it also be possible to document here metadata on specific Items? I'm mainly asking as it would be interesting if we can also provide info around IsAspireProjectResource. cc: @eerhardt 

See [https://github.com/dotnet/aspire/blob/c6d17908dce366b8c08b77509f2ce692b861cbd1/src/Aspire.Hosting.Sdk/SDK/Sdk.targets#L22](https://github.com/dotnet/aspire/blob/c6d17908dce366b8c08b77509f2ce692b861cbd1/src/Aspire.Hosting.Sdk/SDK/Sdk.targets#L22) for more info

Reply:
Nevermind, I see you did this already in AspireHostingSDK